vitrina/datasets/models.py

```diff
@@ -153,6 +153,16 @@ def get_absolute_url(self):
     def get_tag_list(self):
         return list(self.tags.all().values_list('name', flat=True))
 
+    @property
```

Comment:
Properties are supposed to be very light weight, but here is an expensive method, with multiple queries to database. Remove `@property` decorator and make it a method as it should be.